lib/zenaton/services/properties.rb

```diff
@@ -1,7 +1,13 @@
 # frozen_string_literal: true
 
 require 'singleton'
-require 'json/add/core'
+require 'json/add/date'
+require 'json/add/date_time'
+require 'json/add/exception'
+require 'json/add/range'
+require 'json/add/regexp'
+require 'json/add/struct'
+require 'json/add/time'
```

Comment:
requiring `json/add/core` is a shortcut for loading theses files + `json/add/symbol` which this PR removes